src/test/kotlin/uk/gov/dluhc/printapi/messaging/ApplicationRemovedMessageListenerTest.kt

```diff
@@ -31,6 +31,7 @@ internal class ApplicationRemovedMessageListenerTest : IntegrationTest() {
         )
         // currently 29 working days following issue date - refer to application.yml
         val expectedInitialRemovalDate = LocalDate.of(2023, 5, 17)
+        val expectedFinalRemovalDate = LocalDate.of(2032, 7, 1)
```

Comment:
It will nicer to put these 5/7 into months for readability purposes. Although 17th can't be month :)